reconcile/saas_file_owners.py

```diff
@@ -48,6 +46,21 @@ def collect_owners():
                 bot_org_username = bot.get("org_username")
                 if bot_org_username:
                     owners[saas_file_name].add(bot_org_username)
+        # self-service configs
```

Comment:
i'm just duplicating this code since the above one will end up being removed.

Reply:
an economical decision i approve!